pkgs/by-name/un/unicode-paracode/package.nix

```diff
@@ -8,15 +8,17 @@
 }:
 
 python3Packages.buildPythonApplication rec {
+  # We want `pname = "unicode"` so that `nix run nixpkgs#unicode-paracode -- z` runs the `unicode` binary.
```

Comment:
```suggestion
```
This is wrong, please set meta.mainProgram to unicode to achieve the same result.